src/django/api/views.py

```diff
@@ -1473,6 +1475,9 @@ def approve_claim(self, request, pk=None):
 
             send_claim_facility_approval_email(request, claim)
 
+            send_approved_claim_notice_to_list_contributors(request,
```

Comment:
We should consider wrapping this notification in a try block and posting the exception to Rollbar without raising. It may be confusing for the superuser to get a failure message after we successfully committed their approval to the database.

Reply:
Sounds good. This is in 8406351824b27c90638c9b7f28658ad0b3b3c007

I don't know the full range of exceptions that can get thrown when sending an email, so I just used `except Exception` for the try/except blog. In order to be parsimonious about what data get sent to Rollbar I only logged the facility claim id, although I don't know what is included in `sys.exc_info()` so it may be that that needs to go too.

Reply:
This screenshot shows the arguments and local vars that are visible in rollbar when we submit the stack trace. These values are at the item level, not the occurrence level, so they are not deletable. 

![Firefox_Screenshot_2019-06-18T21-35-54 219Z](https://user-images.githubusercontent.com/17363/59721962-587b9100-91d7-11e9-811b-1d260449b365.png)